web/wp-content/themes/borealis-2021/inc/enqueue-scripts-styles.php

```diff
@@ -13,11 +13,37 @@ function pg_mapbox_scripts() {
     }
 
 }
+
+if (!function_exists('threejs_scripts')) {
+    /**
```

Comment:
Small thing, do you mind name-spacing your custom functions, I tend to use 'pg' so the function would be called 'pg_threejs_scripts'. It's a small likelihood, but it makes sure our functions aren't overwritten by WP or any plugins.

Reply:
No problem at all!